src/app/containers/Ad/Canonical/CanonicalAdBootstrapJs.test.jsx

```diff
@@ -9,4 +10,17 @@ describe('CanonicalAds Ads', () => {
       <CanonicalAdBootstrapJs />,
     );
   });
+
+  describe('Assertions', () => {
```

Comment:
Great idea on moving the test to this file. Do they pass now on Jenkins?

Reply:
Yup everything seems stable now! Travis is being a bit weird but I think it's just some issues on their end.

Reply:
Awesome! There are some conflicts that need to be resolved 🔧